### DISCUSSION TITLE
License Conflict

### DISCUSSION THREAD

**Discussion by SkewedZeppelin** [blocklistproject/Lists#670] [Q&A]
This project is licensed under Unlicense, which is a very permissive license.
However these following files are nearly identical (99%) to the proprietary hpHosts files:
```
https://blocklistproject.github.io/Lists/alt-version/malware-nl.txt
https://hosts-file.net/emd.txt

https://blocklistproject.github.io/Lists/alt-version/phishing-nl.txt
https://hosts-file.net/psh.txt

https://blocklistproject.github.io/Lists/alt-version/fraud-nl.txt
https://hosts-file.net/fsa.txt

https://blocklistproject.github.io/Lists/alt-version/ads-nl.txt
https://hosts-file.net/ad_servers.txt
```

See here for numbers:
https://gist.github.com/SkewedZeppelin/cc0f2bba84257c264ceba05a0bad965e

**Comment by SkewedZeppelin** [comment]
@ignoramous you might be interested in this

**Comment by spirillen** [comment]
the host-files was taking down and given as is to the blacklist community, unfortunately I didn't save these words at the time, however I kept a copy of everything else, including the README here https://bitbucket.org/expiredsources/hosts-file.net/src/master/ before it was completely removed by malwarebytes.

**Comment by ignoramous** [comment]
Thanks for the ping.

Folks I spoke to (who weren't lawyers) consider hostfiles to be "documents" and not "code", and hence believe code-specific licenses like APLv2, MPLv2, xGPLs do not cover them: https://github.com/bongochong/CombinedPrivacyBlockLists/issues/14#issuecomment-922482847

Not saying license is violated, because see also (for added confusion): AdroitAdorKhan/EnergizedProtection/issues/46
